deployer.py

```diff
@@ -110,6 +110,31 @@ def stop_bot(bot_name):
     for container in containers:
         for tag in container.image.tags:
             if tag.startswith(bot_name.replace('@', '')):
+                logs = container.logs().decode("utf-8")
+                with open('bots/' + bot_name + '/logs.txt', 'a') as logfile:
```

Comment:
Neat idea storing permanent logs. We'll have to figure out how/when to serve these later.